docs/guides/elasticstack-and-cloud.md

```diff
@@ -67,44 +83,14 @@ provider "elasticstack" {
   }
 }
 
-# Defining a user for ingesting
-resource "elasticstack_elasticsearch_security_user" "user" {
```

Comment:
These resources have been dropped from the generated docs since they were moved to another tf file.